recipes/libuuid/all/conanfile.py

```diff
@@ -1,5 +1,7 @@
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools import files
+from conan.errors import ConanInvalidConfiguration
+from conans import AutoToolsBuildEnvironment, tools
 import os
 
 required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
New tools, new Conan version.